apps/bam.iobio/src/App.tsx

```diff
@@ -96,50 +97,46 @@ function App() {
 		setBamFile(fileStats);
 	}, [fileLoaded]);
 
-	// Init
-	createPercentBox();
-	createHistogram();
-
 	const { mappedReads, forwardStrands, properPairs, singletons, bothMatesMapped, duplicates, histogram } = bamContext;
 
 	return (
 		<div className="App">
-			<header className={'App-header' + (fileLoaded ? ' file-loaded' : ' home')}>
-				<>
-					<img src="images/ov-logo.png" className="App-logo" />
-					<h2>Overture Iobio Components</h2>
-				</>
-				{(fileLoaded && (
+			<header className={clsx('App-header', fileLoaded ? 'file-loaded' : 'home')}>
+				<img src="ov-logo.png" className="App-logo" />
+				<h2>Overture Iobio Components</h2>
+
+				{fileLoaded && (
 					<a className="Back-button" href={'/'}>
 						Back
 					</a>
-				)) || <></>}
+				)}
 			</header>
 			{fileLoaded ? bamFileStats(bamFile) : bamConfigPanel(bamContext, updateContext)}
 
-			<button className={'config-button' + (showBam ? ' active' : '')} onClick={() => toggleShowBam(!showBam)}>
+			<button className={clsx('config-button', showBam && ' active')} onClick={() => toggleShowBam(!showBam)}>
 				Show / Hide BAM
 			</button>
 
-			<div className={'bam-container' + (showBam ? ' bam-open' : '')}>
+			<div className={clsx('bam-container', showBam && 'bam-open')}>
 				{/* Needs to render on the page before scripts for BAM to work */}
 				<div id="app"></div>
 				{showBam ? (
 					<>
 						<h3>Bam.Iobio</h3>
 
-						<iobio-data-broker url="https://s3.amazonaws.com/iobio/NA12878/NA12878.autsome.bam" />
+						<iobio-data-broker alignment-url="https://s3.amazonaws.com/iobio/NA12878/NA12878.autsome.bam" />
```

Comment:
The imports from the `@overture-stack/iobio-components` should be React components, not just webcomponents. The reason for this is that they should include all the state lifecycle management included, to communicate with Arranger and/or their own back end.

Reply:
See PR description
`Component React / Type conversions will be finalized in next PR; this PR uses iobio-chart web-components out of the box`

Reply:
yeah, noticed that after the initial comment, and then edited it to reflect that.